02/app.js

```diff
@@ -20,4 +20,25 @@ function createRandomArray() {
 
 function getRandomInteger(min, max) {
     return Math.round(Math.random() * (max-min) + min);
-}
\ No newline at end of file
+}
+
+//console.log(randomArray.length)
+/*for(let i =0; i<randomArray.length; i++)
+{
+    console.log(randomArray[i])
+    if(randomArray[i] === randomArray[randomArray.length-1])
```

Comment:
tutaj if-a nie powinno być, ostatnie lement należało wyświetlić poza pętlą. Kombinacja `randomArray.length-1` to dobry kierunek